test/cmd/admin.sh

```diff
@@ -172,6 +172,7 @@ os::cmd::expect_success 'oc adm policy add-role-to-user view baz --rolebinding-n
 os::cmd::expect_success_and_text 'oc get rolebinding/view -o jsonpath="{.metadata.name},{.roleRef.name},{.userNames[*]}"' '^view,view,foo$'
 os::cmd::expect_success_and_text 'oc get rolebinding/custom -o jsonpath="{.metadata.name},{.roleRef.name},{.userNames[*]}"' '^custom,view,bar baz$'
 os::cmd::expect_failure_and_text 'oc adm policy add-role-to-user other fuz --rolebinding-name=custom' '^error: rolebinding custom found for role view, not other$'
+os::cmd::expect_success_and_text 'oc adm policy remove-cluster-role-from-group --rolebinding-name=self-provisioners self-provisioner system:authenticated:oauth' "Warning: Your changes may get lost whenever a master is restarted, unless you set the rbac.authorization.kubernetes.io/autoupdate annotation to false for rolebinding self-provisioners"
```

Comment:
Can you change the command to:

`oc adm policy remove-cluster-role-from-group self-provisioner system:authenticated:oauth`

Basically drop the `--rolebinding-name=self-provisioners` because the command should warn without it (and the above is actually what a user would run).